react/features/remote-video-menu/components/web/KickButton.js

```diff
@@ -45,6 +45,7 @@ class KickButton extends AbstractKickButton {
         return (
             <RemoteVideoMenuButton
                 buttonText = { t('videothumbnail.kick') }
+                displayClass = 'kicklink'
                 iconClass = 'icon-kick'
                 id = { `ejectlink_${participantID}` }
```

Comment:
Personally I would not rely on display class, but locate the button by id which consist of the participant id. But it's not a deal breaker for me given we're doing it this way for the other buttons and the PR has been approved by Lenny.

Reply:
What I would like to see one day is usage of data-qa or a similarly named attribute added to the dom so that the tests don't rely on class/id implementation of components. I think we're on the same page though in that I'm okay with the change because it's already the way other button selectors are being done. Also, the id for this button seems excessive because that adds to a stateful dom, plus I think the container for the kick button (the thumbnail) already has an id which can be used to further select down to this link.